packages/front-end/pages/settings/members.tsx

```diff
@@ -0,0 +1,136 @@
+import { FC, useEffect, useState } from "react";
+import { useRouter } from "next/router";
+import InviteList from "@/components/Settings/Team/InviteList";
+import MemberList from "@/components/Settings/Team/MemberList";
+import { useAuth } from "@/services/auth";
+import SSOSettings from "@/components/Settings/SSOSettings";
+import { useUser } from "@/services/UserContext";
+import usePermissions from "@/hooks/usePermissions";
+import { useDefinitions } from "@/services/DefinitionsContext";
+import SelectField from "@/components/Forms/SelectField";
+import OrphanedUsersList from "@/components/Settings/Team/OrphanedUsersList";
+import PendingMemberList from "@/components/Settings/Team/PendingMemberList";
+import { isMultiOrg } from "@/services/env";
+import AutoApproveMembersToggle from "@/components/Settings/Team/AutoApproveMembersToggle";
+import UpdateDefaultRoleForm from "@/components/Settings/Team/UpdateDefaultRoleForm";
+
+const MembersPage: FC = () => {
```

Comment:
Everything in the `pages` directory gets a navigation URL assigned.  It looks like this page is never navigated to directly, just rendered as part of the team page.  If that's the case, move this to `components` instead.